next.config.js

```diff
@@ -15,6 +15,9 @@ module.exports = withBundleAnalyzer({
         ADMIN_GROUPS: config.get("admin.groups"),
         ADMIN_GROUP_ATTRIBUTE: config.get("admin.group_attribute_name"),
         IRODS_HOME_PATH: config.get("irods.home_path"),
+        SESSION_POLL_INTERVAL_MS: config.has("sessions.poll_interval_ms")
+            ? config.get("sessions.poll_interval_ms")
+            : null,
```

Comment:
We can return a `default` value of `5000` here instead of returning null like how we did for resource (CPU, memory etc..) defaults. That would eliminate the changes in `serviceFacades/users.js`.

Reply:
I have to admit that I couldn't get myself to let go of the check for a null value where the configuration was being used. I thought about putting the default value both here and in `serviceFacades/users.js`, but that felt wrong to me. I also thought about defining a default value as a constant here and including this file in `CyVerseAppBar.js`. That felt wrong too. My brain eventually rebelled and I returned null here so that I could let the null check handle the case where the configuration value was undefined. That option seemed the least repulsive to my brain at the time.

As hard as it is for me to do, I think I can convince myself to let go of the null check in `CyVerseAppBar.js`. The upshot is that I'm not just a programmer. I'm a _crazy_ programmer. 🤪

Reply:
Feel free to merge as it is..It was just a suggestion. So no biggie.

Reply:
I think you're right. My brain was just being stubborn yesterday. 😆